### PR TITLE
[daint-gpu] dependencies

### PR DESCRIPTION
**Pull request by jgphpc**
Utility scripts to find dependencies between .ebs

https://madra.cscs.ch/scs/doc/issues/156

### CODE REVIEW

**Comment by jgphpc** [comment]
retest this please

File: jenkins-builds/tools/mf.sh [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+partition=$1
+if [ -z $partition ] ;then
+    usage
+fi
+
+### jenkinslist=$productiondir/jenkins-builds/6.0.UP04-17.08-gpu
+### jenkinslist=$productiondir/jenkins-builds/daint-jg
+#jenkinslist=$productiondir/jenkins-builds/6.0.UP04-17.08-mc
+#jenkinslist=daint-jg
+productiondir=/apps/common/UES/sandbox/jgp/production.git
```
Comment (gppezzi) [inline jenkins-builds/tools/mf.sh]: why do you point to a specific git repository? if this script is being added to the production repository, one would expect that it should use the files on the same repo

**Comment by jgphpc** [comment]
retest this please

File: jenkins-builds/tools/mf.sh [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    #echo "## l0=$l0: $listl1s"
+    echo '------------------------------------------------------------------'
+
+    # iterate through the list of deps'deps (=level2)
+    unset listl2s listl2 l2tmp 
+    for l1 in $listl1b ;do        
+        l2tmp=`grep "^$l1 ->" mf.sh.dot |awk -F"->" '{print $2}' |tr -d \;`
+        listl2=${listl2}$l2tmp
+        listl2s=`echo $listl2 |tr " " "\n" |sort -u |awk '{printf "%s ",$0}'`
+#ok        listofl2deps=`grep "^$l1 ->" mf.sh.dot |awk -F"->" '{print $2}' |tr -d \;`
```
Comment (gppezzi) [inline jenkins-builds/tools/mf.sh]: How about explaining why you keep all these lines commented out?

Comment (jgphpc) [reply jenkins-builds/tools/mf.sh]: d370e2b18c1ffcd88fd76137d4177da9e4dafeff

**Comment by jgphpc** [comment]
eff

**Pull request merged**